R/shiny-getters.R

```diff
@@ -79,31 +80,29 @@
 #'     [addEventListener()](https://github.com/vega/vega/tree/master/packages/vega-view#view_addEventListener)
 #' @export
 #'
-vw_shiny_get_signal <- function(outputId, name, body_value = "value") {
+vw_shiny_get_signal <-
+  function(outputId, name, body_value = "value", id = NULL) {
 
   assert_packages("shiny")
 
   session <- shiny::getDefaultReactiveDomain()
-
-  inputId <- ""
+  ns <- shiny::NS(id)
+  inputId <- glue::glue("{outputId}_signal_{name}")
```

Comment:
I think we should not change the `inputId` logic. If I remember right, this is to avoid namespace collisions if you have more than one call getting the same signal, data, event, etc.

Reply:
Sorry I thought I left a comment about it in this commit, but clearly not. I think the issue I found is that get_unique_inputID will always return inputId unmodified, because make.unique will modify all repeated names except the first. So I think if the aim is to avoid duplicated inputIds we should change the get_unique_inputId function then?